code/client/munkiimport

```diff
@@ -723,6 +728,17 @@ def configure():
         print >> sys.stderr, 'Could not save configuration to %s' % PREFSPATH
 
 
+def init_repo(path):
+    """Initializes a directory with the repo subdirs"""
+    path = os.path.abspath(path)
+    if not os.path.exists(path):
+        os.mkdir(path)
+    for d in REPO_DIRS:
+        os.mkdir(os.path.join(path, d))
+
```

Comment:
I personally would take the traceback rather than an opaque message, but you're probably right. :)